dk.sdu.se4.bullet/src/main/java/dk/sdu/se4/bullet/BulletPlugin.java

```diff
@@ -2,17 +2,22 @@
 
 import dk.sdu.se4.commonbullet.Bullet;
 import dk.sdu.se4.common.entity.Entity;
-import dk.sdu.se4.common.entity.part.DirectionPart;
-import dk.sdu.se4.common.entity.part.ImagePart;
-import dk.sdu.se4.common.entity.part.MovingPart;
-import dk.sdu.se4.common.entity.part.PositionPart;
-import dk.sdu.se4.common.entity.part.TimePart;
+import dk.sdu.se4.common.entity.part.*;
 import dk.sdu.se4.common.service.PluginService;
 import java.io.File;
 
 public class BulletPlugin extends BulletCore implements PluginService {
     @Override
-    public void load() {}
+    public void load() {
+        Entity bullet = new Bullet();
+        bullet.addPart(new SpritePart("Bullets/bullet.png", 16,16,2));
+        bullet.addPart(new PositionPart(-999999, -999999));
+        bullet.addPart(new TimePart(0.01f));
+        bullet.addPart(new DirectionPart(false,false, false, false));
+        bullet.addPart(new MovingPart(0));
+        this.mapService.addEntity(bullet);
```

Comment:
Burde det ikke slettes?

Reply:
nej for den skal køre en entity i opstart for at add spritepart til assetmanageren :)